site/content/entry/yuzu-progress-report-mar-2020/index.md

```diff
@@ -0,0 +1,131 @@
+That’s right, the best way to experience Astral Chain in yuzu is with a small, and typically inferior, iGPU (like the ones that come in any desktop or notebook CPU from Intel).
+
+On the topic of formats, implementing the `RG32` and `RGB32` vertex formats fixed a room that refused to load in The Legend of Zelda: Link’s Awakening in Vulkan.
+Great work [Rodrigo](https://github.com/ReinUsesLisp)!
+
+{{< imgs
+    "./13.png| Works!"
+  >}}
+  
+## Paving the road for even bigger changes
+
+[Rodrigo](https://github.com/ReinUsesLisp)'s initial work on Transform Feedback is finished and it's partially working. 
+This already helps games like POKKÉN Tournament DX, Donkey Kong Country: Tropical Freeze and Xenoblades Chronicles.
+
+An optimization by [Blinkhawk](https://github.com/FernandoS27) reduced the use of RAM by 1GB.
+When talking about RAM usage, less is best!
+
+Now, an important but also astonishing fix; one that nobody expected, not even our project leader [bunnei](https://github.com/bunnei).
+While he was working on fixing what seemed to be a simple bug on Final Fantasy VIII, he also accidentally fixed Super Smash Bros. Ultimate fights!
+This case of serendipity is without a doubt a surprise, but a welcome one.
+
+Now all users can brawl all they want thanks to a measly 5-line change in yuzu’s code!
+However, keep in mind that any Super Smash Bros. Ultimate update newer than 3.1.0 will work, but will also make the characters T-pose, resulting in very funny fights and victory screens.
+We’re working on resolving this, but for now, stick to update 3.1.0 or older!
+
+[bunnei](https://github.com/bunnei) fixed audio crackling in various games like Crash Team Racing and Super Mario Odyssey.
+We are slowly, but surely, improving our audio quality.
+He also implemented initial support for `32-bit ARM` emulation. 
+Nothing tangible can be tested yet, but thanks to this, it will be possible in the future to run `32-bit` games like Mario Kart 8 Deluxe.
+
+And last but not least, [Blinkhawk](https://github.com/FernandoS27) has been making some serious changes to the yuzu's GPU emulation, while also laying the foundation for the recent rework of yuzu's Virtual Memory Manager (codename: Project Epimetheus).
+
+These, in turn, will help further advance the development of Project Prometheus; one of the super-secret Projects our devs are currently working on to bring some huge changes to yuzu’s performance and hardware requirements.
+Stay tuned to hear more about them in the future!
+
+That’s all for now folks, see you on the April report.
+
+&nbsp;
+<h4 style="text-align:center;">
+<b>Please consider supporting us on [Patreon](https://www.patreon.com/yuzuteam)!<br>
+If you would like to contribute to this project, checkout our [GitHub](https://github.com/yuzu-emu/yuzu)!</b>
```

Comment:
```suggestion
If you would like to contribute to this project, check out our [GitHub](https://github.com/yuzu-emu/yuzu)!</b>
```

Reply:
That's our default footnote, I don't want to edit it.

Reply:
Liu is correct, checkout should be `check out`. Can't believe we never caught that lol